tests/test_download_utils.py

```diff
@@ -207,3 +208,10 @@ def test_download_tar_file(mocker, mock_file, mock_untar):
     mock_untar.assert_called_once_with("foo", cleanup=True)
     if os.path.exists('a'):
         shutil.rmtree('a')
+
+def test_extractall_unicode(mocker,mock_file,mock_unzip):
```

Comment:
Could you add an additional test file that has content which breaks the previous version?

Reply:
I do not have access to a Windows computer at the moment but I will try to find out one to create a similar archive to the IRMAS one

Reply:
done!